near/omni-prover/wormhole-omni-prover-proxy/src/parsed_vaa.rs

```diff
@@ -135,7 +135,8 @@ struct FinTransferWh {
     token_address: OmniAddress,
     amount: u128,
     recipient: String,
```

Comment:
Could we rename it to make it more consistent?

```suggestion
    fee_recipient: String,
```